app/helpers/contents_list_helper.rb

```diff
@@ -1,7 +1,13 @@
 module ContentsListHelper
   def wrap_numbers_with_spans(content_item_link)
     content_item_text = strip_tags(content_item_link) #just the text of the link
-    number = /^[0-9]*[.]/.match(content_item_text)
+
+    # Must start with a number
+    # Number must be between 1 and 999 (ie not 2014)
+    # Must be followed by a space
+    # May contain a period `1.`
+    # May be a decimal `1.2`
+    number = /^\d{1,3}(\.?|\.\d{1,2})(?=\s)/.match(content_item_text)
```

Comment:
Correct, that was intentional. I didn't find any examples of `X.ABC`. I felt like it should be limited. Something like 1.0003 might reasonably be a decimal rather than a heading level. Unlikely to happen in reality.